PUBLISHING.md

```diff
@@ -4,87 +4,99 @@
   </h1>
 
   <h3 align="center">
-    Current release owner 🎉⛸️🦆 <a href='https://github.com/leblowl'>@leblowl</a> (February 2024)
+    Current release owner ✨<a href='https://github.com/adrastaea'>@adrastaea</a>✨ (September 2024)
   </h3>
 
   <br />
   <br />
 </p>
 
-## Trigger
-Release process begins when all issues from `Sprint` are merged and moved to `Ready for QA`.  
+## Release Trigger
 
-## Branching strategy
-Each release starts with it's own branch (based on develop).
-Develop becomes a draft for the next release.
-From this moment, all the fixes (patches) for the last supported version are being merged into the release branch and cherry-picked into develop.
+- [ ] [Quiet Planning Board](https://github.com/orgs/TryQuiet/projects/3) is up to date
+- [ ] All issues to be included from `Sprint` are merged and moved to `Merged`
+- [ ] `Ready for QA` column is empty of issues from previous releases
 
-### Example:
-```
+## Branching Rules
+
+1. Release branches are based on the `develop` branch.
+1. Release branches are named after the version number, e.g. `2.1.0` not the pre-release version number, e.g. `2.1.0-alpha.1`.
+1. Once a release branch is created, it is frozen and no new features are to be merged into it. Only bug fixes are allowed.
+1. Release branches are never deleted.
+1. Any hotfixes for the release branch are merged into the release branch and then cherry-picked into the `develop` branch if necessary.
+
+```plaintext
+# Example of branching strategy
 /develop  ->      /2.1.0       ->  #patch-commit
           ->  #feature-commit  ->  #patch-commit (cherry-picked)  -> /2.2.0
                                                                   -> ...
 ```
 
+## Release Flow
+
+1. Prepare a release candidate (alpha).
+1. Deliver the alpha to QA.
+1. QA tests the alpha.
+1. If QA finds issues, a github issue is created and moved to the `Sprint` column. Any fixes are merged into the release branch and `develop` branch.
+1. If QA finds no issues, the release is approved.
+1. A production release is built.
+1. QA tests the production release.
+1. Production release is published.
+
+## Checklist for Alpha Release
+
+- [ ] Release branch is created from `develop` branch with the production version number, e.g. `2.1.0`.
+- [ ] Reviewed the base `CHANGELOG.md` file and ensured that it is up to date with all changes included in the release since the last production release. Package level `CHANGELOG.md` files are automatically updated during the release process.
+- [ ] Approved the updated `CHANGELOG.md` file with the @holmesworcester.
```

Comment:
We can cut this since it repeats later. 

The main idea here is that CHANGELOG is a public facing document that we want to be clear, readable, and meaningful, and that conforms more or less to the style described in publishing guidelines.

Reply:
I'm just going to remove the approval part from the alpha release section because I think its valuable for QA to have an updated changelog for the alpha too